DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/CourseContent/_EditSectionDiagnostic.cshtml

```diff
@@ -2,7 +2,7 @@
 @using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.CourseContent
 @model EditCourseSectionViewModel
 
-<div class="nhsuk-grid-column-one-half">
+<div class="nhsuk-grid-column-one-half" @(Model.ShowDiagnostic ? "" : "hidden")>

```

Comment:
I think it would make more sense to do the hiding of this partial where it is called rather than inside the partial itself?